src/extension/rn-extension.ts

```diff
@@ -103,7 +103,7 @@ function setupAndDispose<T extends ISetupableDisposable>(setuptableDisposable: T
 function warnWhenReactNativeVersionIsNotSupported(): void {
     return reactNativeProjectHelper.validateReactNativeVersion().done(() => { }, reason => {
         TelemetryHelper.sendSimpleEvent("unsupportedRNVersion", { rnVersion: reason });
-        const shortMessage = `React Native Tools only supports React Native versions 0.19.0 and later`;
+        const shortMessage = `React Native Tools extension only supports React Native versions 0.19.0 and later`;
```

Comment:
Given that we'll show the message if the react-native package is not installed I'd change this to something like:
React Native Tools extension can't work without React Native 0.19.0 or later on node_modules/react-native.